discord/ext/commands/core.py

```diff
@@ -508,9 +568,9 @@ async def transform(self, ctx, param):
             argument = view.get_quoted_word()
         view.previous = previous
 
-        return await run_converters(ctx, converter, argument, param)
+        return await run_converters(ctx, converter, argument, param)  # type: ignore
```

Comment:
Why the `# type: ignore` here?

Reply:
type-checker believes argument could still be none, would need to cast.